src/components/DataBar/DataBar.stories.tsx

```diff
@@ -68,7 +68,7 @@ export const NoSegments: StoryObj<Args> = {
 
 const Interactive = () => {
   const [max, setMax] = React.useState(100);
-  const handleChange = (e) => {
+  const handleChange = (e: any) => {
```

Comment:
Nit - I'd put this directly on the element

```tsx
<input
  onChange={(e) => { ... }}
/>
```

and then you get the types for free, and without writing any types. Probably better than using `any`, in any case (which sort of defeats the purpose)